client/components/common/tce-core/QuestionContainer/Controls.vue

```diff
@@ -11,13 +11,20 @@
         Save
       </v-btn>
     </div>
-    <v-btn
-      v-else
-      @click="$emit('edit')"
-      color="blue-grey darken-4"
-      text large>
-      Edit
-    </v-btn>
+    <div v-else>
+      <v-btn
```

Comment:
You can't put it here. tcc assessment pool is a custom container and container specifics need to be ha ndled within it. Let's discuss.